cmake/projects/Boost/schemes/url_sha1_boost_ios_library.cmake.in

```diff
@@ -345,7 +348,7 @@ ExternalProject_Add(
     INSTALL_COMMAND
     "${CMAKE_COMMAND}" -E make_directory "@HUNTER_PACKAGE_INSTALL_PREFIX@/lib"
     COMMAND
-    "@HUNTER_SELF@/scripts/create-boost-ios-fatlib.sh"
+    "@HUNTER_GLOBAL_SCRIPT_DIR@/create-boost-ios-fatlib.sh"
```

Comment:
> very sorry for this stupid mistake

Actually it's a "good" mistake. Now I see that new testing system I've introduced is effective :)